src/grt/src/GlobalRouter.cpp

```diff
@@ -92,7 +92,6 @@ void GlobalRouter::init()
   _verbose = 0;
 
   // Clock net routing variables
-  _pdRev = 0;
```

Comment:
omg, it never made sense to set a pointer to 0.

Reply:
`_pdRev` wasn't a pointer.

Reply:
oh, it was a bool. that makes even less sense to use zero